src/downloader/sentry_address.rs

```diff
@@ -0,0 +1,16 @@
+use http::Uri;
```

Comment:
use `url::Url` instead

Reply:
the reason I've used http crate is because tonic Endpoint uses it already, and it generates `SentryClient::connect(endpoint)` that is possible to call with Uri;

this doesn't actually introduce a new dependency

if you want url::Url, we could introduce it later